traffic_ops/testing/api/utils/utils.go

```diff
@@ -119,7 +130,7 @@ type V3TestData struct {
 
 // V3TestDataT represents the data needed for testing the v3 api endpoints.
 type V3TestDataT[B any] struct {
-	EndpointId     func() int
+	EndpointID     func() int
```

Comment:
I agree it creates a bigger diff and had removed. But since I had reviewed it multiple times already, I asked him to add it to give continuity to my review.

Reply:
Even if you reviewed it and keeping it lends continuity to your review, that doesn't make it have to do with the feature added in the PR itself. It does not belong in this PR.

Reply:
I understand but removing at this stage will be a bigger effort than necessary. So, IMO, I would let it be in hopes that this is not repeated in future.

Reply:
Removing it at this stage is not difficult, it's a single *refactor* command per field in an IDE

Reply:
It's not just my personal opinion, it's the guidelines that our community voted on. Under [*One pull request per feature*](https://github.com/apache/trafficcontrol/blob/master/CONTRIBUTING.md#one-pull-request-per-feature):

> Like big functions, big pull requests are just hard to review. Make each pull request as small as possible.

There's a reason we have that guideline, and it's for cases like these.

Reply:
I have nothing more to add since I get where you are coming from..  each developer tries and does scope creep.. some are small and we let it go.. some are big (like the current one) which is hard to review..  I have made peace with a big diff on the PR and I'm ok to let it (scope creep) go this time. I would be very grateful if you let it too.

Reply:
> I would be very grateful if you let it too.

Nope

Reply:
It's not just a little change. Reverting it brings *Files changed* for this PR down from 160 to 100.

Reply:
Correction: Brings it down from 160 to 60

Reply:
Removing is very simple (if somewhat annoying because I already did that and then un-did it) but I don't think that work fully captures what Rima means by "it would be more work at this point to remove it". It doesn't belong in the PR, that's true, but it's already been reviewed as a part of this PR. It was a _simple_ change, but not a _small_ one. So to pull that out into a separate PR is duplicating work already spent on reviewing those changes here, which isn't insignificant. Having already volunteered yourself for the task, it's clear you're prepared to take on that tedium, but consider that the time it took Rima to review it already will have been wasted, which is why she asked me to put it back in after I took it out.

At this point I've put it in, taken it out, and put it back in again. I truly do not care if it goes in this PR, but if I change it again I want to be assured it'll be the last time. Perhaps get a third reviewer to break the tie. Or don't if you can come to a consensus without that. But I just need to know what I have to do.

Reply:
I'm fine with you keeping the 100-file field rename in #7099. Separately, I've removed those so I could see what the API tests diff looks like without that part, and it looks like #7099 is missing test coverage for the `"PRIMED"` DeliveryServiceActiveState. #7099 is also missing coverage for an invalid DeliveryServiceActiveState being POSTed or PUT.

Reply:
> ...it looks like https://github.com/apache/trafficcontrol/pull/7099 is missing test coverage for the "PRIMED" DeliveryServiceActiveState.

I see two Delivery Services in the fixtures data that use that active state, so it seems to be covered to me. It's also worth noting that all Delivery Services with `active: false` in APIv3 and APIv4 tests are implicitly using "PRIMED"; technically the new state is "INACTIVE", not "PRIMED".

> ... missing coverage for an invalid DeliveryServiceActiveState being POSTed or PUT.

That's true. Ideally it should have both.

Reply:
Thank you @zrhoffman for being ok with a big scope creep in this PR.